week-1-ts-ecosystem/w01e01-typescript-basics/src/utils/has-given-age.ts

```diff
@@ -0,0 +1,5 @@
+import { User } from "types";
+
+export function hasGivenAge(requiredAge: number): (user: User) => boolean {
+  return (user) => user.age >= requiredAge;
+}
```

Comment:
Nazwa `utils` do niej się u wszystkich doczepiam :) 

Jeśli już to nie wiem,,, jakieś `validators`

Tutaj ciekawie, pierwszy raz widzę rozdzielenie tych 2 validatorów per pliki.
No i wydaje mi sie to ciekawe podejście. Fajnie